tests/test_parcels.py

```diff
@@ -486,8 +485,19 @@ def test_tree_buffers_null(self, features_gdf, parcels_gdf):
             calc_buffers=True,
             primary_decision="largest_intersection",
         )
-        # For this test, every result should be NaN for buffers, as there's always a building overlapping the parcel boundary.
-        assert df.filter(like="buffer").head().isna().all().all()
+        # For this test, every parcel has buildings that cross the edges.
+        # One parcel has the primary roof not intersecting the edge:
+        assert not df.loc["1_0"].filter(like="buffer_0ft").isna().any()
+        assert not df.loc["1_0"].filter(like="buffer_5ft").isna().any()
+        assert df.loc["1_0"].filter(like="buffer_10ft").isna().all()
+        assert df.loc["1_0"].filter(like="buffer_30ft").isna().all()
+        assert df.loc["1_0"].filter(like="buffer_100ft").isna().all()
+
+        assert df.loc["1_0", "primary_roof_area_sqm"] == pytest.approx(df.loc["1_0", "primary_roof_buffer_0ft_zone_sqm"], abs=0.05)
+        assert df.loc["1_0", "primary_roof_area_sqm"] == pytest.approx(df.loc["1_0", "primary_roof_buffer_0ft_roof_sqm"], abs=0.05)
+
+        # every other result should be NaN for buffers, as the primary building is always overlapping the parcel boundary.
+        assert df.filter(like="buffer").notna().any(axis=1).sum() == 1
```

Comment:
This is summing to `2` for me and not `1`

Reply:
Yes, on manual checks, parcel 3_3 now gets retained as a multiparcel building, and clipped to parcel boundary. That means it can get a valid 0ft buffer (but not 5ft etc). Updating test case.